src/components/auth/RegistrationForm.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <form
+    class="registration-form"
+    @submit.prevent="register(user)"
+  >
+    <InputCustom
+      label="username"
+      v-model="user.username"
+      :required="true"
+      @input="clearMessage"
+    />
+    <InputCustom
+      label="e-mail"
+      type="email"
+      v-model="user.email"
+      :required="true"
+      @input="clearMessage"
+    />
+    <InputCustom
+      label="password"
+      type="password"
+      v-model="user.password"
+      :required="true"
+    />
+    <InputCustom
+      label="repeat password"
+      type="password"
+      v-model="passwordToCheck"
+      :required="true"
+    />
+    <div class="message-container">
+      <div
+        class="response-message"
+        v-if="responseMessage.length"
+      >
+        {{ responseMessage }}
+      </div>
+      <div
+        class="error-message"
+        v-if="errorMessage.length"
+      >
+        {{ errorMessage }}
+      </div>
+      <div
+        class="error-message"
+        v-if="arePasswordsEntered && !isPasswordConfirmed"
+      >
+        wrong password
+      </div>
```

Comment:
there is a lot of html here.
If there is gonna be in the same place - maybe we can use different one div with optional classes?